pandas/core/internals/array_manager.py

```diff
@@ -741,7 +741,13 @@ def insert(self, loc: int, item: Hashable, value, allow_duplicates: bool = False
 
         value = extract_array(value, extract_numpy=True)
         if value.ndim == 2:
-            value = value[0, :]
+            if value.shape[0] == 1:
```

Comment:
np.atleast_2d?

Reply:
It's actually converting a 2D array to 1D, so `atleast_2d` is not useful here (and eg `squeeze` also not, because that would also convert a len-1 array to a scalar, which we don't want there)